documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -133,3 +133,22 @@ The date MAY optionally be followed by one of:
 * 2016-07-04-rc0 - Just use "rc", not "rc" + number
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
+
+### <a name="M3016" />M3016 PropertiesNamesCamelCase
+**Output Message**: Property named: "{0}", must follow camelCase style. Example: "{1}".
+**Output Message**: Property named: "{0}", for definition: "{1}" must follow camelCase style. Example: "{2}".
+
+**Description**: Property names must use lowerCameCase style.  
+
+**Good Examples**: Examples of lowerCamelCase style:
+* camelCase
+* resourceKey
+* fooBarBaz
+
+**Bad Examples**: The following would be invalid:
+* PascalCase
+* UpperCamelCase
+* alllowercase
+* miXeDcApItAlIzAtIoN
```

Comment:
@RobDolinMS The guidelines I was pointed to at https://msdn.microsoft.com/en-us/library/141e06ef(v=vs.71).aspx indicate that acronyms of more than 2 letters should follow camel casing, so in your example it should be someApiEndpoint.